config/repositories.yml

```diff
@@ -14,10 +14,10 @@ ars:
      <div class="al-repository-contact-info"><a href="mailto:soundarchive@stanford.edu">soundarchive@stanford.edu</a></div>
   thumbnail_url: 'repositories/ars.png'
   url: https://library.stanford.edu/libraries/archive-recorded-sound
-  # request_types:
-  #   aeon_web_ead:
-  #     request_url: 'https://stanford.aeon.atlas-sys.com/logon'
-  #     request_mappings: 'Action=10&Form=31&Value=ead_url'
+  request_types:
```

Comment:
If desired I believe you could initiate a variable here:
```
  request_types: &default_aeon_request
    aeon_web_ead:
      request_url: 'https://stanford.aeon.atlas-sys.com/logon'
      request_mappings: 'Action=10&Form=31&Value=ead_url'
```

And then down the line use 
```
  request_types: *default_aeon_request
```

Reply:
This is a good idea. But I still think a change would be needed upstream in Arclight to allow this. Alias parsing is has to be explicitly enabled when you parse the yaml and we're not doing that: https://github.com/projectblacklight/arclight/blob/8274ec83720a8fc46e34022cc54ee4faa63e6beb/lib/arclight/repository.rb#L75